python/galois/lonestar/analytics/deg_count.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
This is already in master. Please rebase against master so it doesn't show up here.